File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Orders/IOrderData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Orders;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IOrderData extends IOrdertTotalPrice,IOrderNote,IOrderStore,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Orders/IOrderData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Orders
{
    public interface IOrderData :  CoreObject.IObjectID, CoreObject.IObjectTitle, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IOrderTotalPrice, IOrderStore, IOrderNote
    {

    }
}
